client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-import-or-migrate/index.tsx

```diff
@@ -22,74 +23,49 @@ const SiteMigrationImportOrMigrate: Step = function ( { navigation, flow } ) {
 	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
 	const { deleteMigrationSticker } = useMigrationStickerMutation();
 	const { mutate: cancelMigration } = useMigrationCancellation( site?.ID );
-
-	const isBusinessPlan = site?.plan?.product_slug
-		? isWpComBusinessPlan( site?.plan?.product_slug )
-		: false;
-
-	let options;
-
+	const siteCanInstallPlugins = canInstallPlugins( site );
+	const isUpgradeRequired = ! siteCanInstallPlugins;
 	const isMigrationExperimentEnabled = useMigrationExperiment( flow );
 
-	if ( isMigrationExperimentEnabled ) {
-		const badgeText = isBusinessPlan
-			? translate( 'Included with your plan' )
-			: // translators: %(planName)s is a plan name (e.g. Commerce plan).
-			  ( translate( 'Available on %(planName)s with 50% off', {
+	const options = useMemo( () => {
+		const upgradeRequiredLabel = isMigrationExperimentEnabled
+			? translate( 'Available on %(planName)s with 50% off', {
+					args: { planName: getPlan( PLAN_BUSINESS )?.getTitle() ?? '' },
+			  } )
+			: translate( 'Requires %(planName)s plan', {
 					args: { planName: getPlan( PLAN_BUSINESS )?.getTitle() ?? '' },
-			  } ) as string );
+			  } );
 
-		options = [
-			{
-				label: translate( 'Migrate site' ),
-				description: translate(
+		const migrateOptionDescription = isMigrationExperimentEnabled
+			? translate(
 					"Best for WordPress sites. Seamlessly move all your site's content, themes, plugins, users, and customizations to WordPress.com."
-				),
-				value: 'migrate',
-				badge: {
-					type: 'info-blue' as BadgeType,
-					text: badgeText,
-				},
-				selected: true,
-			},
-			{
-				label: translate( 'Import content only' ),
-				description: translate( 'Import posts, pages, comments, and media only.' ),
-				value: 'import',
-			},
-		];
-	} else {
-		options = [
+			  )
+			: translate( "All your site's content, themes, plugins, users and customizations." );
+
+		return [
 			{
 				label: translate( 'Migrate site' ),
-				description: translate(
-					"All your site's content, themes, plugins, users and customizations."
-				),
+				description: migrateOptionDescription,
 				value: 'migrate',
 				badge: {
 					type: 'info-blue' as BadgeType,
-					// translators: %(planName)s is a plan name (e.g. Commerce plan).
-					text: translate( 'Requires %(planName)s plan', {
-						args: { planName: getPlan( PLAN_BUSINESS )?.getTitle() ?? '' },
-					} ) as string,
+					text: isUpgradeRequired ? upgradeRequiredLabel : translate( 'Included with your plan' ),
 				},
 				selected: true,
 			},
 			{
 				label: translate( 'Import content only' ),
-				description: translate( 'Import just posts, pages, comments and media.' ),
+				description: translate( 'Import posts, pages, comments, and media only.' ),
```

Comment:
why is this copy changed?

Reply:
Unexpected AI changes 🤦 
Removed here [2d8c438](https://github.com/Automattic/wp-calypso/pull/98017/commits/2d8c438f5db333c9cdd1e19706b5a454992c1267)